README.md

```diff
@@ -79,36 +78,45 @@ adyen.checkout.version = 50
 - payment_session
 - payments.result
 - payment_methods
+- payment_methods.balance
 - payments
 - payments.details
 - payment_links
-
-**checkout utility:**
+- payment_links.get
+- payment_links.update
 - origin_keys
+- orders
+- orders.cancel
 
 **payments:**
 - authorise
 - authorise3d
+- authorise3ds2
+- get_authentication_result
+- retrieve3ds_result
 
 **modifications:**
 - capture
 - cancel
 - refund
 - cancel_or_refund
+- technical_cancel
 - adjust_authorisation
 - donate
+- void_pending_refund
 
 **payouts:**
 - confirm_third_party
 - decline_third_party
 - store_detail
 - submit_third_party
 - store_detail_and_submit_third_party
+- payout
 
 **recurring:**
 - list_recurring_details
 - disable
-- store_token
```

Comment:
Why are we removing /storeToken?  I know it's generally discouraged, but there are definitely merchants who use it.

Reply:
Let's leave it in.  Not sure if any merchants who use the library use this endpoint, so would be safer to not make any breaking changes without investigating.